### ISSUE TITLE
Provide service-manager config options for HTTP proxy settings

### ISSUE THREAD

**Issue by praveenkumar** [projectatomic/vagrant-service-manager#302]
Since we are working with proxy issue and it would be nice that service-manager have variable to consume it like it have for openshift image, tag and docker registry.

**Comment by budhrg** [comment]
@praveenkumar Where are you working?

**Comment by brgnepal** [comment]
Changing milestone as proxy feature will not be part of upcoming ADB release.

**Comment by hferentschik** [comment]
Proxy feature is not part of ADB/CDK, right?

**Comment by brgnepal** [comment]
As we are trying to minimize the shipped Vagrantfile content, enabling proxy through will allow users to quickly enable/disable for proxy environment.

Example of Vagrant file changes:

```
config.servicemanager.http_proxy = 'proxy-url'
config.servicemanager.http_proxy_user = 'proxy-user'
config.servicemanager.http_proxy_password = 'proxy-password'
```

This will potentially remove the following lines from [Openshift Vagrantfile](https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/components/rhel/rhel-ose/Vagrantfile):

```
# explicitly enable and start OpenShift
config.vm.provision "shell", run: "always", inline: <<-SHELL
    PROXY=#{PROXY} PROXY_USER=#{PROXY_USER} PROXY_PASSWORD=#{PROXY_PASSWORD} /usr/bin/sccli openshift
SHELL
```

with need an update in line 

```
  config.servicemanager.services = "docker"
```

to

```
  config.servicemanager.services = "docker, openshift"          # or openshift is enough
```

**Comment by hferentschik** [comment]
Resolved via pull request #400
